lib/analyzer/package-managers/apt.ts

```diff
@@ -72,7 +101,14 @@ function parseDpkgLine(
       curPkg.Version = value;
       break;
     case "Source":
-      curPkg.Source = value.trim().split(" ")[0];
+      const parts = value.trim().split(" ");
+      curPkg.Source = parts[0];
+      if (parts.length > 1) {
+        curPkg.SourceVersion = parts[1]
+          .trim()
+          .replace("(", "")
+          .replace(")", "");
+      }
```

Comment:
A comment explaining what the value might look like and what we need to extract would be helpful here.

I understand what it's doing, but I don't understand what the outcome of it is :) (I can guess, but an example would make it easier)